src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page-content">
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image-wrapper">
+        <img
+          src="./images/imac.jpeg"
+          alt="photo APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="product-card__image"
+        >
+      </div>
+
+      <h2 class="product-card__product-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) 
+      </h2>
+
+      <p class="product-card__product-code">
+        Product code: 195434
+      </p>
+
+      <div
+        class="product-card__rewiev-wrapper"
+      >
```

Comment:
It can in 1 line but it's up to you
```suggestion
      <div class="product-card__rewiev-wrapper">
```